reference/test/stop/combined.cpp

```diff
@@ -134,7 +146,7 @@ TEST_F(Combined, WaitsTillTime)
     double time_d = std::chrono::duration_cast<double_seconds>(time).count();
 
 
-    ASSERT_GE(time_d + eps, 1.0e-9);
+    ASSERT_GE(time_d, 1.0e-2);
```

Comment:
This should have been `timelimit` previously, and it should be that now.
You could also change the `eps` definition depending on Windows / Linux.